images/virt-artifact/patches/015-rename-core-resources.patch

```diff
@@ -51,38 +14,27 @@ index 228518871..55ce72b6c 100644
  		},
  		Spec: policyv1.PodDisruptionBudgetSpec{
  			MinAvailable: &minAvailable,
-diff --git a/pkg/virt-operator/resource/generate/components/crds.go b/pkg/virt-operator/resource/generate/components/crds.go
-index 822f3d82b..36126ef43 100644
---- a/pkg/virt-operator/resource/generate/components/crds.go
-+++ b/pkg/virt-operator/resource/generate/components/crds.go
-@@ -862,7 +862,7 @@ func NewKubeVirtPriorityClassCR() *schedulingv1.PriorityClass {
- 			Kind:       "PriorityClass",
- 		},
- 		ObjectMeta: metav1.ObjectMeta{
--			Name: "kubevirt-cluster-critical",
-+			Name: "kubevirt-internal-virtualization-cluster-critical",
- 		},
- 		// 1 billion is the highest value we can set
- 		// https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass
 diff --git a/pkg/virt-operator/resource/generate/components/deployments.go b/pkg/virt-operator/resource/generate/components/deployments.go
-index 4d00a423a..ced56e776 100644
+index 4d00a423a..6b6e40920 100644
 --- a/pkg/virt-operator/resource/generate/components/deployments.go
 +++ b/pkg/virt-operator/resource/generate/components/deployments.go
-@@ -166,7 +166,7 @@ func newPodTemplateSpec(podName, imageName, repository, version, productName, pr
+@@ -166,9 +166,8 @@ func newPodTemplateSpec(podName, imageName, repository, version, productName, pr
  			Name: podName,
  		},
  		Spec: corev1.PodSpec{
 -			PriorityClassName: "kubevirt-cluster-critical",
-+			PriorityClassName: "kubevirt-internal-virtualization-cluster-critical",
- 			Affinity:          podAffinity,
- 			Tolerations:       criticalAddonsToleration(),
+-			Affinity:          podAffinity,
+-			Tolerations:       criticalAddonsToleration(),
++			Affinity:    podAffinity,
++			Tolerations: criticalAddonsToleration(),
  			Containers: []corev1.Container{
-@@ -529,7 +529,7 @@ func NewOperatorDeployment(namespace, repository, imagePrefix, version, verbosit
+ 				{
+ 					Name:            podName,
+@@ -529,7 +528,6 @@ func NewOperatorDeployment(namespace, repository, imagePrefix, version, verbosit
  					Name: VirtOperatorName,
  				},
  				Spec: corev1.PodSpec{
 -					PriorityClassName:  "kubevirt-cluster-critical",
```

Comment:
There are patches for all deployments in int...virt...kubevirts/config, I think we can leave `PriorityClassName` as-is to reduce this patch. WDYT?